cli/flags.rs

```diff
@@ -948,6 +950,13 @@ Ignore linting a file by adding an ignore comment at the top of the file:
 fn repl_subcommand<'a, 'b>() -> App<'a, 'b> {
   runtime_args(SubCommand::with_name("repl"), false, true)
     .about("Read Eval Print Loop")
+    .arg(
+      Arg::with_name("eval")
+        .long("eval")
+        .help("Evaluates the provided code when the REPL starts.")
+        .takes_value(true)
+        .value_name("code"),
```

Comment:
I've noticed some parts of the code require an equals sign. Should this? `deno repl --eval="console.log(5);"`? I'm not sure when to use the equals sign and when not to.

Reply:
I think it shouldn't require equals. The equals sign is required in some places to disambiguate. I don't remember the exact situation off hand, but we should attempt to remove the equals-sign-requirement if the ambiguity has evaporated.